prac_06/guitar.py

```diff
@@ -0,0 +1,22 @@
+# Guitar
+"""
+Guitar
+Estimate: 15 minutes
+Actual:   10 minutes
+"""
+YEAR =  2024
```

Comment:
CURRENT_YEAR  or PRESENT_YEAR would better suit as your talking about the current year to determine the age